public/index.html

```diff
@@ -186,6 +186,10 @@
       fetchBItems()
     </script>
 
+    <!-- start model-viewer -->
+    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
```

Comment:
maybe this could be async
https://www.w3schools.com/tags/att_script_defer.asp

else this JS parses every time the page loads.

Even better would be to call `useScript` when we want it